src/sagas/flavor.js

```diff
@@ -38,22 +36,140 @@ function* requestStashWorker() {
       throw new Error('Request failed for an unspecified reason!');
     }
   } catch (error) {
-    // eslint-disable-next-line
-    console.dir(error);
-    yield put(actions.requestStashFailure(error));
+    yield put(actions.requestFailure(error));
+  }
+}
+
+function* addStashWorker({ flavor }) {
+  try {
+    const user = yield call(getCurrentUser);
+
+    const endpoint = {
```

Comment:
Should have a check for null user or a `user?.id` to prevent a ReferenceError

Reply:
getCurrentUser() throws an error if user is not truthy. I added a check for user.id though, just in case.